docs/tutorial/index.md

```diff
@@ -0,0 +1,12 @@
+# Tutorial: Extending Platform UI with new navigation
```

Comment:
@mere-teresa, I combined index and "extending_platformui" into one file.
I think a nav menu is not necessary, as this will (soon) be integrated in the new doc page with a decent navigation.